### PR TITLE
Redesigned ListenService

### PR DESCRIPTION
**Pull request by 07jasjeet**
### ONLY MERGE AFTER #136 

# Problem
1) [MOBILE-127](https://tickets.metabrainz.org/browse/MOBILE-127)
2) Listen was only submitted when the previous song was paused, then played and then skipped to desired song. Listens were not submitted at all if songs were played continuously without break.
# Solution
Redesign with timer to keep seconds in count and removed all the confusing code before.

### CODE REVIEW

**Review by akshaaatt — APPROVED** [review]
LGTM! Thanks for the update 💯

File: app/src/main/java/org/listenbrainz/android/model/ListenSubmitBody.kt [hunk 1]
```diff
@@ -26,7 +26,7 @@ class ListenSubmitBody {
     }
 
     private fun Payload.setClientDetails(): Payload{
-        this.metadata.additionalInfo.submission_client = BuildConfig.APPLICATION_ID
+        this.metadata.additionalInfo.submission_client = "ListenBrainz Android"
```
[the remark this answers is not in the document]
Comment (07jasjeet) [reply app/src/main/java/org/listenbrainz/android/model/ListenSubmitBody.kt:29]: All the scrobblers are submitting listens in this format, so I though so be it.

[the remark this answers is not in the document]
Comment (akshaaatt) [reply app/src/main/java/org/listenbrainz/android/model/ListenSubmitBody.kt:29]: I don't think we should change the format now. How would the previous listens identify then? We don't want to create confusion.

[the remark this answers is not in the document]
Comment (07jasjeet) [reply app/src/main/java/org/listenbrainz/android/model/ListenSubmitBody.kt:29]: App still in beta so I think it should be fine no?

[the remark this answers is not in the document]
Comment (07jasjeet) [reply app/src/main/java/org/listenbrainz/android/model/ListenSubmitBody.kt:29]: Currently it says 'org.listenbrainz.android'
Other apps like pano scrobbler and web scrobbler submit like this: 
'Pano Scobbler' and 'Web Scrobbler'

[the remark this answers is not in the document]
Comment (07jasjeet) [reply app/src/main/java/org/listenbrainz/android/model/ListenSubmitBody.kt:29]: But we could sure reverse it no major problem anyways

[the remark this answers is not in the document]
Comment (akshaaatt) [reply app/src/main/java/org/listenbrainz/android/model/ListenSubmitBody.kt:29]: We are live on the playstore @07jasjeet . Please revert this change for now.

[the remark this answers is not in the document]
Comment (akshaaatt) [reply app/src/main/java/org/listenbrainz/android/model/ListenSubmitBody.kt]: Thanks!

**Pull request merged**